composer.json

```diff
@@ -14,14 +14,15 @@
   "require-dev": {
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
     "phpcompatibility/php-compatibility": "^9.3",
-    "phpunit/phpunit": "^8.5",
+    "phpunit/phpunit": "^3|^4|^5|^6|^7|^8|^9",
     "squizlabs/php_codesniffer": "^3.5",
     "wp-coding-standards/wpcs": "^2.2",
     "wp-phpunit/wp-phpunit": "^5.8",
     "yoast/phpunit-polyfills": "^1.0"
   },
   "require": {
-    "composer/installers": "~1.0"
+    "composer/installers": "~1.0",
+    "php": "^5.2.4|^5.3|^5.4|^5.5|^5.6|^7|^8"
```

Comment:
I think we don't need to check below 5.6. So here it should be `>=5.6`

Reply:
Thank you for your feedback, I have adjusted the code now